docs/user-guide/tiramisu/browser-exploit.md

```diff
@@ -8,6 +8,7 @@ Make sure your Wii U has internet access for this step.
 
 1. Take the SD Card out of your computer and plug it into your Wii U console.
 1. Launch the Internet Browser and navigate to the website `wiiuexploit.xyz`.
-1. Click on `Run Exploit!` and hold the B button until you see a menu, this will be necessary for the next steps.
+1. Click on `Run Exploit!` and hold the B button until you see a screen that says `Choose a payload:`, this will be necessary for the next steps.
     - If your Wii U gets stuck on a white or otherwise frozen screen, wait a few seconds. If nothing happens, reboot the console, [reset the browser's save data](https://en-americas-support.nintendo.com/app/answers/detail/a_id/1507/~/how-to-delete-the-internet-browser-history) and try again.
+    - If you see a blue menu with the options `Tiramisu` and `Installer`, you did not hold down B long enough. Try again.
```

Comment:
As suggested on the other pull request, I think that "Tiramisu and Installer" should just be replace with "EnvironmentLoader". ie:
```suggestion
    - If you see EnvironmentLoader, you did not hold down B long enough. Try again.
```

Reply:
That would be a better idea